StandardAnalysis/test/test_tensorflow_MC2022_cfg.py

```diff
@@ -474,3 +474,145 @@
+process.tensorflowProducer.inputTensorNameDS = cms.string("input")
+process.tensorflowProducer.inputTrackTensorNameDS = cms.string("input_track")
+process.tensorflowProducer.outputTensorNameDS = cms.string("model_1/output_xyz/Softmax")
+
+process.tensorflowProducer.triggers = cms.InputTag("TriggerResults", "", "HLT")
+process.tensorflowProducer.triggerObjects = cms.InputTag("slimmedPatTrigger")
+process.tensorflowProducer.genParticles = cms.InputTag("prunedGenParticles", "")
+process.tensorflowProducer.met = cms.InputTag("slimmedMETs")
+process.tensorflowProducer.electrons = cms.InputTag("slimmedElectrons", "")
+process.tensorflowProducer.muons = cms.InputTag("slimmedMuons", "")
+process.tensorflowProducer.taus = cms.InputTag("slimmedTaus", "")
+process.tensorflowProducer.pfCandidates = cms.InputTag("packedPFCandidates", "")
+process.tensorflowProducer.vertices = cms.InputTag("offlineSlimmedPrimaryVertices", "")
+process.tensorflowProducer.jets = cms.InputTag("slimmedJets", "")
+
+process.tensorflowProducer.rhoCentralCalo = cms.InputTag("fixedGridRhoFastjetCentralCalo")
+
+process.tensorflowProducer.EBRecHits = cms.InputTag("reducedEcalRecHitsEB")
+process.tensorflowProducer.EERecHits = cms.InputTag("reducedEcalRecHitsEE")
+#process.tensorflowProducer.ESRecHits = cms.InputTag("reducedEcalRecHitsES")
+process.tensorflowProducer.HBHERecHits = cms.InputTag("reducedHcalRecHits", "hbhereco")
+#process.tensorflowProducer.CSCSegments = cms.InputTag("cscSegments")
+#process.tensorflowProducer.DTRecSegments = cms.InputTag("dt4DSegments")
+#process.tensorflowProducer.RPCRecHits = cms.InputTag("rpcRecHits")
+
+process.tensorflowProducer.dEdxPixel = cms.InputTag ("dedxPixelHarmonic2", "")
+process.tensorflowProducer.dEdxStrip = cms.InputTag ("dedxHarmonic2", "")
+process.tensorflowProducer.isolatedTracks = cms.InputTag("isolatedTracks", "")
+process.tensorflowProducer.isoTrk2dedxHitInfo = cms.InputTag("isolatedTracks", "")
+#process.tensorflowProducer.genTracks = cms.InputTag("generalTracks", "")
+process.tensorflowProducer.pileupInfo = cms.InputTag ("addPileupInfo")
+
+process.tensorflowProducer.minTrackPt = cms.double(20.0)
+process.tensorflowProducer.minGenParticlePt = cms.double(-1)
+process.tensorflowProducer.maxRelTrackIso = cms.double(-1)
+process.tensorflowProducer.dataTakingPeriod = cms.string("2022")
+process.tensorflowProducer.etaRangeNearTrack = cms.double(-1)
+process.tensorflowProducer.phiRangeNearTrack = cms.double(-1)
+process.tensorflowProducer.maxNumOfRecHits = cms.int32(-1)
+process.tensorflowProducer.signalTriggerNames = cms.vstring([
+        'HLT_MET105_IsoTrk50_v',
+        'HLT_PFMET120_PFMHT120_IDTight_v',
+        'HLT_PFMET130_PFMHT130_IDTight_v',
+        'HLT_PFMET140_PFMHT140_IDTight_v',
+        'HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_v',
+        'HLT_PFMETNoMu130_PFMHTNoMu130_IDTight_v',
+        'HLT_PFMETNoMu140_PFMHTNoMu140_IDTight_v',
+        'HLT_PFMET250_HBHECleaned_v',
```

Comment:
We are not using the `HBHECleaned` paths anymore. Maybe this should be updated with the paths in the trigger config file

Reply:
this is actually junk now that the OSUGenericTrack producer handles this, I would remove the tensorflow producers if we don't need it

Reply:
As discussed in MM, these can be removed in this PR, then

Reply:
removed